npm/create-greenjs-app/main.js

```diff
@@ -23,17 +23,42 @@ function copyFolderRecursiveSync(source, target) {
     }
 }
 
```

Comment:
Need a review on this file, which adds the --template flag.

The rest of the changes are new templates added, and the existing templates added to the npm folder.